anger.js

```diff
@@ -26,15 +26,16 @@ function anger (opts) {
     ? identifier
     : (payload) => get(payload, identifier)
 
-  for (var i = 0; i < clients.length; i++) {
+  for (let i = 0; i < clients.length; i++) {
     clients[i] = new Client(opts.url)
     if (i < opts.senders) {
       senders[i] = clients[i]
     }
   }
 
+  let i = 0
   steed.each(clients, (client, done) => {
-    client.connect({ auth: auth }, done)
+    client.connect({ auth: auth(client, i++) }, done)
```

Comment:
If `each()` is for concurrent execution, is it guaranteed to call this in the same order as the array above? Probably but just checking :)